config/initializers/session_store.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_coursemology2_session'
+Rails.application.config.session_store :cookie_store, key: '_coursemology2_session',
+                                                      same_site: :strict
```

Comment:
Apparently this didn't do anything on staging, but nginx-side could configure the SameSite: Strict settings. can revert this or leave it, it doesn't seem to affect.